Documentation/Concepts/Cache.rst

```diff
@@ -0,0 +1,111 @@
+*  In certain time intervals, usually once a day
+*  If content with a starttime or endtime set becomes visible or invisible
+*  In some other cases that your integrator or developer might have defined
+
+Scenario: Changes are not visible in the frontend
+==================================================
+
+If the cache did not get automatically cleared, the changes you made might not
+be visible right away in the frontend.
+
+Some scenarios in which manual cache clearing might be necessary:
+
+*  You changed the title of a page. You view the affected page and everything
+   looks fine. Now you go to the start page and cannot see your change in the
+   menu. The following happened: Changing a pages title deletes the cache of the
+   affected page. It does, however, not delete the cache of other pages like the
+   start page.
+
+*  You added a news record. It does not appear in the news list. Creating a
+   record does not automatically delete the cache of any page unless the
+   integrator made some configurations.
+
+*  You changed content on one page that should also be displayed on other pages.
+   This is often done with static parts of the page that should be editable
+   nevertheless such as footer, a carousel in the header, etc.
+
+Manual cache clearing
+=====================
+
+Clearing the cache of a single page can be done without ill effect in most
+installations.
+
+To clear the cache of that page, go to the module :guilabel:`Web > Page`
+and click the button representing a lightening bolt with the tooltip
+:guilabel:`Clear cache for this page`.
+
+.. figure:: /Images/ManualScreenshots/Cache/ClearPageCache.png
+   :alt: Clear cache for this page
+
+   Clear cache for this page
+
+When you reload the affected page, the changes should be visible right away.
+If the changes are not visible, also clear the :ref:`browser-cache`. Or use an
+incognito tab in your browser.
+
+Only editors who received the according user rights can flush the cache of
+all pages. If you cannot see the lightening bold button beside the help menu
+button in the toolbar you might not have the right to do that:
+
+.. figure:: /Images/ManualScreenshots/Cache/ClearAllPageCaches.png
+   :alt: Flush cache of all pages
+
+   Flush cache of **all** pages
+
+.. warning:: After flushing the cache each page has to be generated a new
+   on the first request for that page. This requires resources on the server
+   such as processor and hard disc usage. If too many pages have to be
+   generated at the same time, generating them might take longer (maybe minutes
+   instead of seconds) or fail all together. Therefore some TYPO3 installations
+   do not allow manual cache flushing. Talk to you Administrator about the
+   preferred strategy here.
+
+Previewing a page without clearing the cache
+=============================================
+
+If you need to preview a page and cannot clear its cache you can use the
```

Comment:
The no_cache parameter can be disabled in LocalConfiguration.php - so this should be reflected:
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Configuration/DefaultConfiguration.php#L1346

Reply:
Added a sentence about it, editors do not need to know the technical details but only that the administrator might have disabled this